src/Routes/Sign-In/Sign-in.jsx

```diff
@@ -59,12 +59,14 @@ export function SignIn (){
       <form onSubmit={handleSubmit} action='' className='flex flex-col'>
         <Input name='Email' id='email' type='email' value={email} onChange={(e) => setEmail(e.target.value)} placeholder='Write your email here!'/>
         <Input name='Password' id='password' type='password' value={password} onChange={(e) => setPassword(e.target.value)} placeholder='Write your password here!'/>
-        {error && <p className="text-red-500 flex items-center w-11/12 m-auto md:pl-20">{error}</p>}       
+        {error && <p className="text-red-500 flex items-center w-11/12 m-auto md:pl-20">{error}</p>}
         <input type="submit" value="Sign in" className='m-auto w-32 h-10 border border-slate-50 border-solid rounded-lg transition-all ease-in-out duration-100 md:text-lg md:hover:bg-red-800' />
       </form>
       <div className='flex flex-col items-center gap-1 mt-6'>
         <small>Don't you have an account?</small>
-        <small>Click here and get a new one!</small>
+        <small>
+          Click <Link to='/recover-password'><span className='underline decoration-dotted decoration-white'>here</span></Link> and get a new one!
```

Comment:
The "don't you have an account? Click here and get a new one!" should be redirect to the "sign up" page? 
because it is being redirected to the recover password, in this case I think the text could be changed to say: "do you forgot your password? Click here and get a new one"